core/src/main/java/jenkins/views/JenkinsHeader.java

```diff
@@ -0,0 +1,44 @@
+package jenkins.views;
+
+import java.util.List;
+import java.util.logging.Logger;
+import java.util.stream.Collectors;
+
+import org.kohsuke.accmod.Restricted;
+import org.kohsuke.accmod.restrictions.NoExternalUse;
+
+import edu.umd.cs.findbugs.annotations.CheckForNull;
+import hudson.Extension;
+import hudson.ExtensionList;
+
+/**
+ * Default {@link Header} provided by Jenkins
+ * 
+ * @author Ildefonso Montero
+ * @see Header
+ */
+@Extension(ordinal = Integer.MIN_VALUE)
+public class JenkinsHeader implements Header {
+    
+    private static Logger LOGGER = Logger.getLogger(JenkinsHeader.class.getName());
+
+    @Override
+    public boolean isHeaderEnabled() {
+        return true;
+    }
+    
+    @Restricted(NoExternalUse.class)
+    @CheckForNull
+    public static Header get() {
```

Comment:
Don't this method belongs to the `Header` interface (which should be an abstract class then)? 
`JenkinsHeader` should be _just_ another extension, which happens to be always there.

Reply:
Sounds good. Updated in https://github.com/jenkinsci/jenkins/pull/5909/commits/ae940851ff1ec57f2d214a35d3626e8280fac23e. I let in the `JenkinsHeader` before given it was the default implementation of the `Header` provided by the core, but yes, I prefer to include it there and to refactor as an abstract class instead of an interface.